frontend/src/layout/navigation/TopBar/AccountPopover.tsx

```diff
@@ -223,12 +226,16 @@ export function AccountPopoverOverlay(): JSX.Element {
                 {isCloudOrDev ? (
                     <LemonButton
                         onClick={closeAccountPopover}
-                        to={urls.organizationBilling()}
+                        to={
+                            featureFlags[FEATURE_FLAGS.BILLING_USAGE_DASHBOARD]
+                                ? urls.organizationBillingOverview()
+                                : urls.organizationBilling()
```

Comment:
There are quite a few instances of this call so I think it might be better to change in the function

 E.g.
```
    organizationBilling: (products?: ProductKey[]): string =>
	featureFlags[FEATURE_FLAGS.BILLING_USAGE_DASHBOARD] 
		? `/organization/billing/overview${products && products.length ? `?products=${products.join(',')}` : ''}`,
	        : `/organization/billing${products && products.length ? `?products=${products.join(',')}` : ''}`,
  ```

Reply:
I only found one instance of this, so I don't know if it's a good idea to create a function for this now. Maybe in the future?

Reply:
I see 12 instances and it will also make it so if anyone adds another this this doesn't get missed. Change one place instead of 12.
<img width="394" alt="Screenshot 2024-12-27 at 10 25 08 AM" src="https://github.com/user-attachments/assets/943f4e01-d534-4457-b49b-98b3c56744a4" />

Reply:
Ahhhh I haven't thought in change those yet. I was looking for instances where the `if feature flag` was already implemented. Ok, going to do this.